app/views/profiles/_edit_public.haml

```diff
@@ -1,10 +1,4 @@
--#   Copyright (c) 2010-2011, Diaspora Inc.  This file is
--#   licensed under the Affero General Public License version 3 or later.  See
--#   the COPYRIGHT file.
-
 - content_for :head do
-  = javascript_include_tag :profile
```

Comment:
`jquery.autoSuggest.custom` was before in that file which we already load in `main.js` and `mobile/mobile.js`.